### ISSUE TITLE
How I can preview both Front and Rear camera?

### ISSUE THREAD

**Issue by qhu91it** [pinkfish/flutter_rtmppublisher#41]
I want use both camera feeds at the same time in my app, have any option for that?

**Comment by pinkfish** [comment]
No.  Not sure how possible that is?  I think the OS in most cases only lets
you use one camera?

**Comment by qhu91it** [comment]
Hi, it like this article on Android: [How to Use the Front and Rear Camera Previews Simultaneously in Android Using the Camera2 API](https://medium.com/better-programming/simultaneous-front-and-rear-camera-preview-in-android-using-camera2-api-d9f2eb9af6b5)
and for iOS (>=13) is new feature Multi-Camera.
